code/index.html

```diff
@@ -16,7 +16,7 @@
 
       <img
         class="guess-who-icon"
-        src="images/guess-who-bubble.png"
+        src="images/SeekPng.com_guess-logo-png_1405949.png"
```

Comment:
I like the official logo for the game you used and matching color palette for the rest of the game!